ERCS/erc-7562.md

```diff
@@ -293,6 +293,13 @@ Alt-mempools are served by the same bundlers participating in the canonical memp
 * **[AREP-020]** the alt-mempool becomes THROTTLED based on the [Reputation Calculation](#reputation-calculation)
 * **[AREP-030]** REMOVED
 
+### EIP-7702 Authorizations
```

Comment:
remove "7702" from title: we could have auth-related issued unrelated to 7702 in the future.